tests/update/updates_packagekit_gpk.pm

```diff
@@ -77,7 +77,8 @@ sub run {
             if (match_has_tag("updates_none")) {
                 send_key 'ret';
                 if (check_screen "updates_installed-restart", 0) {
-                    power_action 'reboot';
+                    select_console 'root-sonsole';
+                    type_string "reboot\n";
```

Comment:
I am not sure if this is the right approach. @okurz already told you to use **power_action**. Maybe the function **power_action** needs to be adapted to also care of this use case?

Reply:
That should be the next step, yes.
But I think its more important to get it running first and then fix cosmetic stuff afterwards
@okurz whats you opinion on this?